Add codewind entry to new project wizard

jopit opened the pull request:
Signed-off-by: John Pitman <jspitman@ca.ibm.com>

keithchong commented on dev/src/main/java/org/eclipse/codewind/intellij/module/CodewindModuleBuilder.java:
```diff
@@ -0,0 +1,130 @@
+        String projectType = template.getProjectType();
+        String conid = "local";
+        try {
+            // Codewind won't create a project in a non-empty directory, and by this
```
Is there something in Codewind that can be done?  eg. instead of non-empty directories, can they relax that restriction and at least ignore all .* files ?

keithchong commented on dev/src/main/java/org/eclipse/codewind/intellij/module/TemplateTableModel.java:
```diff
@@ -0,0 +1,72 @@
+
+    @Override
+    public int getColumnCount() {
+        return 3;
```
Perhaps add a comment here to say there are only 3 columns here and it is fixed?

keithchong commented on dev/src/main/resources/messages/ui/CodewindUIBundle.properties:
```diff
@@ -358,3 +358,11 @@ RepoUpdateTask=Updating template sources
 RepoListErrorTitle=Template Source List Error
 RepoListErrorMsg=An error occurred while trying to get the list of template sources: {0}
+
+CodewindNotStarted=Codewind must be running in order to create projects or modules.
```
Maybe ask ID to review the new strings?

keithchong commented on dev/src/main/java/org/eclipse/codewind/intellij/module/InstallCodewindStep.java:
```diff
@@ -0,0 +1,76 @@
+                CoreUtil.openDialog(CoreUtil.DialogType.INFO, message("CodewindLabel"), message("CodewindInstalledStarted"));
+            } else {
+                // ProgressManager.getInstance().run(new InstallCodewindTask(this::onInstall));
+                CoreUtil.openDialog(CoreUtil.DialogType.INFO, message("CodewindLabel"), "Not yet implemented");
```
Add to message?

keithchong commented on dev/src/main/java/org/eclipse/codewind/intellij/module/CodewindModuleBuilder.java:
```diff
@@ -0,0 +1,130 @@
+        } catch (Exception e) {
+            Logger.logWarning(e);
+        }
+        return null;
```
I can't recall the state of the wizard buttons in your demo and whether there was a Previous button there.   What happened if you press previous and next in succession?

jopit commented:
@cccanderson, I think I've addressed all your comments.  I've left the copyright dates as 2019, because that's the year those files were initially created.  Please review the changes and let me know what you think.
